library/ws-widget/collection/src/lib/btn-page-back/btn-page-back.component.ts

```diff
@@ -47,8 +48,15 @@ export class BtnPageBackComponent extends WidgetBaseComponent
     public router: Router,
     private configSvc: ConfigurationsService,
     private discussUtilitySvc: DiscussUtilsService,
+    private translate: TranslateService,
+    private langtranslations: MultilingualTranslationsService
   ) {
     super()
+    if (localStorage.getItem('websiteLanguage')) {
+      this.translate.setDefaultLang('en')
```

Comment:
Please use 'en' from constant file, because in future may we have to set default language as per region